pkg/helm/controller.go

```diff
@@ -279,15 +284,19 @@ func job(chart *helmv1.HelmChart) (*batch.Job, *core.ConfigMap, *core.ConfigMap)
 		})
 	}
 
+	job.Spec.Template.Spec.NodeSelector = make(map[string]string)
+	job.Spec.Template.Spec.NodeSelector[core.LabelOSStable] = "linux"
```

Comment:
Does the k8s package have a const for this value?

Reply:
No, just lots of string literals like this one.

Reply:
Darn, was hoping for something like they have for namespaces. Oh well. Thanks!